Apps/WebClient/src/NewClientApp/src/components/private/dependent/AddDependentComponent.vue

```diff
@@ -0,0 +1,457 @@
+                                        id="dateOfBirth"
+                                        v-model="dependent.dateOfBirth"
+                                        label="Date of Birth"
+                                        data-testid="dateOfBirthInput"
+                                        :max-date="maxBirthdate"
+                                        :error-messages="
+                                            ValidationUtil.getErrorMessages(
+                                                v$.dependent.dateOfBirth
+                                            )
+                                        "
+                                        @blur="
+                                            v$.dependent.dateOfBirth.$touch()
+                                        "
+                                    />
+                                </div>
+                            </v-col>
+                            <v-col cols="12" sm="6">
+                                <label for="phn">PHN</label>
+                                <div>
+                                    <v-text-field
+                                        id="phn"
+                                        v-model="dependent.PHN"
+                                        v-maska:[phnMaskaOptions]
+                                        label="PHN"
+                                        clearable
+                                        type="text"
+                                        data-testid="phnInput"
+                                        :error-messages="
+                                            ValidationUtil.getErrorMessages(
+                                                v$.dependent.PHN
+                                            )
+                                        "
+                                        @blur="v$.dependent.PHN.$touch()"
+                                    />
+                                </div>
+                            </v-col>
+                        </v-row>
+                        <v-row>
+                            <v-col>
+                                <v-checkbox
+                                    id="termsCheckbox"
+                                    v-model="accepted"
+                                    data-testid="termsCheckbox"
+                                    density="compact"
+                                    color="primary"
+                                    class="text-body-1 checkbox-top"
+                                    hide-details
+                                    :error="
+                                        !ValidationUtil.isValid(v$.accepted)
+                                    "
+                                    @blur="v$.accepted.$touch()"
+                                >
+                                    <template #label>
+                                        <div class="ml-1">
```

Comment:
NIT: I'll leave this to your discretion:

I doubt the div is really necessary here too, similar to above, p is a block element and should take the entire width already. The margin class is unusual as the checkbox styling should already be managed by vuetify.

Reply:
No change.  Div makes a difference.

<img width="736" alt="Screenshot 2023-07-28 at 2 14 29 PM" src="https://github.com/bcgov/healthgateway/assets/58790456/76c7603b-9ede-4ff0-b5cb-d530e77beec4">